wrapper/src/main/java/software/amazon/jdbc/PropertyDefinition.java

```diff
@@ -53,6 +53,11 @@ public class PropertyDefinition {
       new AwsWrapperProperty(
           "wrapperPlugins", null, "Comma separated list of connection plugin codes");
 
+  public static final AwsWrapperProperty IS_AUXILIARY_CONNECTION =
```

Comment:
Should it be a public configuration parameter?

Reply:
I was wondering about that as well, but this parameter is accessed in multiple packages (util.connection and plugin). I can add a sentence in the description to clarify that it should not be used by users, and we can keep it out of our documents, but is there a better solution?